addons/l10n_it_edi/models/account_edi_format.py

```diff
@@ -661,23 +715,18 @@ def _import_fattura_pa(self, tree, invoice):
                             natura_element = element.xpath('.//Natura')
                             invoice_line_form.tax_ids.clear()
                             if percentage is not None:
+                                tax_extra_domain = buyer_seller_info[incoming]['tax_extra_domain']
```

Comment:
Rename to `type_tax_use_domain`, it's more explicit.
At the moment we don't see other `extra` uses for that key in the dictionary.